web/src/main/java/com/futurewei/alcor/web/rbac/aspect/RbacConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *
+ * Copyright 2019 The Alcor Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *         you may not use this file except in compliance with the License.
+ *         You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *         Unless required by applicable law or agreed to in writing, software
+ *         distributed under the License is distributed on an "AS IS" BASIS,
+ *         WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *         See the License for the specific language governing permissions and
+ *         limitations under the License.
+ * /
+ */
+
+package com.futurewei.alcor.web.rbac.aspect;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(value = {"com.futurewei.alcor.common.rbac", "com.futurewei.alcor.web.rbac.aspect"})
+public class RbacConfiguration {
+}
```

Comment:
Do we plan to have some fun with RbacConfiguration? It is empty for now. :-)

Reply:
This file is config @ComponentScan Only, So it have empty body.